CollapseLauncher/Classes/GameManagement/GameSettings/Universal/RegistryClass/CollapseMiscSetting.cs

```diff
@@ -0,0 +1,76 @@
+﻿using CollapseLauncher.Interfaces;
+using Hi3Helper;
+using Hi3Helper.EncTool;
+using Microsoft.Win32;
+using System;
+using System.Text;
+using System.Text.Json;
+using static CollapseLauncher.GameSettings.Base.SettingsBase;
+using static Hi3Helper.Logger;
+
+namespace CollapseLauncher.GameSettings.Universal
+{ 
+    internal class CollapseMiscSetting : IGameSettingsValue<CollapseMiscSetting>
+    {
+        #region Fields
+        private const string _ValueName = "CollapseLauncher_Misc";
+        #endregion
+
+        #region Properties
+        /// <summary>
+        /// This defines if the game's main process should be run in Above Normal priority.<br/><br/>
+        /// Default: false
+        /// </summary>
+        public bool UseGameBoost { get; set; } = false;
+        #endregion
+
+        #region Methods
+#nullable enable
+        public static CollapseMiscSetting Load()
+        {
+            try
+            {
+                if (RegistryRoot == null) throw new NullReferenceException($"Cannot load {_ValueName} RegistryKey is unexpectedly not initialized!");
```

Comment:
Instead of checking for a null value, you can catch multiple exception:

`try {
	some code here
} 
catch (Exception ex) {}
catch (NullReferenceException ex2) {some code here}`